### PR TITLE
[5_15] Update Develop_on_Linux.md for Fedora

### PR DESCRIPTION
**Pull request by erentar**
## Why you open this Pull Request?

Add instructions for fedora in the development documentation

## What work have you done in the current Pull Request?

Complete installation in docker and note down the required commands

### CODE REVIEW

File: docs/guide/Develop_on_Linux.md [hunk 1]
```diff
@@ -10,6 +10,15 @@ sudo apt update
 sudo apt install --yes build-essential libfontconfig1-dev qtbase5-dev libqt5svg5-dev
 ```
 
+On Fedora and RHEL derivatives:<!-- (tested on docker) -->
+```
+sudo dnf update
+sudo dnf groupinstall "C Development Tools and Libraries"
+sudo dnf install git wget unzip glibc-static libstdc++-static qt5-qtbase-devel qt5-qtsvg-devel fontconfig-devel
+wget https://xmake.io/shget.text -O - | bash #install xmake
+git clone https://github.com/XmacsLabs/mogan
```
Comment (darcy-shen) [inline docs/guide/Develop_on_Linux.md:19]: how to git clone this repo, and cd into the directory is not necessary for developers

File: docs/guide/Develop_on_Linux.md [hunk 2]
```diff
@@ -10,6 +10,15 @@ sudo apt update
 sudo apt install --yes build-essential libfontconfig1-dev qtbase5-dev libqt5svg5-dev
 ```
 
+On Fedora and RHEL derivatives:<!-- (tested on docker) -->
```
Comment (darcy-shen) [inline docs/guide/Develop_on_Linux.md:13]: Remove `tested on docker`. Are you using Fedora as a desktop?

File: docs/guide/Develop_on_Linux.md (hunk 1 above)
Comment (darcy-shen) [reply docs/guide/Develop_on_Linux.md:19]: it is wildly known knowledge

Comment (erentar) [reply docs/guide/Develop_on_Linux.md:19]: Sure i'll remove those

File: docs/guide/Develop_on_Linux.md (hunk 2 above)
Comment (erentar) [reply docs/guide/Develop_on_Linux.md:13]: I do but i listed the packages needed from a fresh install

**Pull request merged**